test/tools/ossfuzz/config/solidity.dict

```diff
@@ -0,0 +1,210 @@
+"continue"
+"delete "
+"do"
+"else"
+"for (a=0;a<2;a++)"
+"function bid() public payable {"
+"modifier onlySeller() {"
+"if"
+"new "
+"return"
+"switch"
+"throw"
+"try"
+"while"
+"pragma solidity ^90.90.0"
+"pragma experimental ABIEncoderV2;"
+"pragma experimental SMTChecker;"
+"pragma solidity >=0.4.0 <0.6.0;"
+"x << y"
+"x * 2**y"
+"x / 2**y"
+"int x = -2**255;"
+"x % y"
+"0**0"
+"<<"
+">>"
+"|"
+"&"
+"^"
+"~"
+"<="
+"<"
+"=="
+"!="
+">="
+">"
+"+"
+"-"
+"*"
+"/"
+"%"
+"**"
+"fixed "
+"unfixed "
+"ufixed128x128 "
+"fixed128x128 "
+"float "
+"double "
+".balance"
+".transfer("
+".send("
+"true "
+"false "
+".call("
+".callcode("
+".delegatecall( "
+".gas("
+".value("
+"this"
+"2e10"
+"1.1"
+"hex"001122FF""
+"bool "
+"byte "
+"bytes1 "
+"bytes2 "
+"bytes3 "
+"bytes4 "
+"bytes5 "
+"bytes6 "
+"bytes7 "
+"bytes8 "
+"bytes9 "
+"bytes10 "
+"bytes11 "
+"bytes12 "
+"bytes13 "
+"bytes14 "
+"bytes15 "
+"bytes16 "
+"bytes17 "
+"bytes18 "
+"bytes19 "
+"bytes20 "
+"bytes21 "
+"bytes22 "
+"bytes23 "
+"bytes24 "
+"bytes25 "
+"bytes26 "
+"bytes27 "
+"bytes28 "
+"bytes29 "
+"bytes30 "
+"{ "
+"} "
+"("
+")"
+"public "
+"contract "
+"struct V {"
+"enum B {"
+"returns "
+"uint "
+"uint256 "
+"uint8 "
+"int "
+"int256 "
+"int8 "
+"bool "
+"return "
+"for"
+"if"
+"constructor "
+"internal "
+"external "
+"pure "
+"constant "
+"view "
+"payable "
+"bytes32 "
+"library l {"
+"uint[] "
+"uint[][5] "
+"[a, b, c]"
+"length "
+"push "
+"pop "
+"++"
+"--"
+"storage sto = "
+"memory m ="
+"string str = "
+"//"
+";"
+"blockhash("
+"block.blockhash("
+"block.coinbase("
+"block.difficulty("
+"block.gaslimit("
+"block.number("
+"block.timestamp("
+"gasleft("
+"msg.data("
+"msg.gas("
+"msg.sender("
+"msg.sig("
+"msg.value("
+"now("
+"tx.gasprice("
+"tx.origin("
+" wei"
+" finney"
+" szabo"
+" ether"
+" seconds"
+" minutes"
+" hours"
+" days"
+" weeks"
+" years"
+"abi.encode("
+"abi.encodePacked("
+"abi.encodeWithSelector("
+"abi.encodeWithSignature("
+"assert("
+"require("
+"require(msg.sender == seller,\"\""
```

Comment:
Perhaps better to use `== 0`?

Reply:
Do you mean adding a new token called `"== 0"` or `"require(msg.sender == 0,\"\""`?